src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,18 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
```

Comment:
(this isn't your fault) but all of these lines from the @Override to the second last closing brace should be moved over 1 space

see: https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation